tools/wiktionary2lex/parsers/parser_lib.py

```diff
@@ -32,3 +33,19 @@ def print_output(output):
                 print key + ":" + list_item[key] + "\t",
         print "\n"
 
+def make_entry(word,pronunciation,POS,language,accent,xsampa):
```

Comment:
Can this be make_entries (makes it clearer that multiple entries may be generated and explains why it is a list)